README.md

```diff
@@ -113,7 +113,7 @@ For more information, see the [wiki](https://github.com/Azure/azure-storage-fuse
 
 - General options
     * `accountName`: Specifies the storage account blobfuse targets.
-    * `blobEndpoint`: Specifies the blob endpoint to use. Defaults to *.blob.core.windows.net, but is useful for targeting storage emulators.
+    * `blobEndpoint`: Specifies the blob endpoint to use. Defaults to *.blob.core.windows.net, but is useful for targeting storage emulators. (e.g. storageaccountname.blob.region.azurestack.company.com)
```

Comment:
the writing should " Optional parameter, if it is not specified defaults to blob.core.windows.net". This parameter should be specified for zonal dns accounts,  custom domain name accounts or storage emulators.